elfeed-search.el

```diff
@@ -360,12 +360,16 @@ The customization `elfeed-search-date-format' sets the formatting."
                                title-width
                                elfeed-search-title-max-width)
                         :left)))
-    (insert (propertize date 'face 'elfeed-search-date-face) " ")
-    (insert (propertize title-column 'face title-faces 'kbd-help title) " ")
-    (when feed-title
-      (insert (propertize feed-title 'face 'elfeed-search-feed-face) " "))
-    (when tags
-      (insert "(" tags-str ")"))))
+    (let ((beg (point)))
+      (insert (propertize date 'face 'elfeed-search-date-face) " ")
+      (insert (propertize title-column 'face title-faces 'kbd-help title) " ")
+      (when feed-title
+	(insert (propertize feed-title 'face 'elfeed-search-feed-face) " "))
+      (when tags
+	(insert "(" tags-str ")"))
+      (add-text-properties
```

Comment:
You should probably use `put-text-property` here, since you're only setting one property.